flutter_stetho crashed after updating to version 0.3.0

Hey, i just updated the package to version 0.3.0.

Unfortunately, it crashes on build with the following exception:

```
Launching lib\main.dart on WAS LX1A in debug mode...
Initializing gradle...
Resolving dependencies...
Running Gradle task 'assembleDebug'...
registerResGeneratingTask is deprecated, use registerGeneratedResFolders(FileCollection)
registerResGeneratingTask is deprecated, use registerGeneratedResFolders(FileCollection)
registerResGeneratingTask is deprecated, use registerGeneratedResFolders(FileCollection)

Compiler message:
file:///C:/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_stetho-0.3.0/lib/src/http_client_response.dart:4:7: Error: 'StethoHttpClientResponse' can't implement both 'Stream<List<int>>' and 'Stream<Uint8List>'
 - 'Stream' is from 'dart:async'.
 - 'List' is from 'dart:core'.
 - 'Uint8List' is from 'dart:typed_data'.
class StethoHttpClientResponse extends StreamView<List<int>>
      ^
Compiler failed on C:\Dev\petleo-flutter-app\lib\main.dart

FAILURE: Build failed with an exception.

* Where:
Script 'C:\Dev\flutter\packages\flutter_tools\gradle\flutter.gradle' line: 652

* What went wrong:
Execution failed for task ':app:compileflutterBuildDebugArm64'.
> Process 'command 'C:\Dev\flutter\bin\flutter.bat'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 12s
Finished with error: Gradle task assembleDebug failed with exit code 1
```

Channel dev, v1.8.1, (I also tried beta 1.7) 
Dart 2.5.0 (build 2.5.0-dev.0.0 b5aeaa6796)


Any ideas how to solve this problem?


Best
Junus

--- a/lib/src/http_client_response.dart
+++ b/lib/src/http_client_response.dart
@@ -1,11 +1,12 @@
 import 'dart:async';
 import 'dart:io';
+import 'dart:typed_data';
 
-class StethoHttpClientResponse extends StreamView<List<int>>
+class StethoHttpClientResponse extends StreamView<Uint8List>
     implements HttpClientResponse {
   final HttpClientResponse response;
 
-  StethoHttpClientResponse(this.response, Stream<List<int>> stream)
+  StethoHttpClientResponse(this.response, Stream<Uint8List> stream)
       : super(stream);
 
   @override

--- a/lib/src/utils.dart
+++ b/lib/src/utils.dart
@@ -1,12 +1,13 @@
 import 'dart:async';
 import 'dart:io';
 import 'dart:math';
+import 'dart:typed_data';
 
 import 'package:flutter_stetho/src/method_channel_controller.dart';
 
 /// Create a response transformer that can intercept and pipe the http response
 /// data to the Method channel
-StreamTransformer<List<int>, List<int>> createResponseTransformer(String id) {
+StreamTransformer<Uint8List, Uint8List> createResponseTransformer(String id) {
   return new StreamTransformer.fromHandlers(handleData: (data, sink) {
     sink.add(data);
     MethodChannelController.onData({"data": data, "id": id});

